### PR TITLE
Requests

### PR DESCRIPTION
**Pull request by pradeephubgit**
Since I faced issues when trying to complete a GitHub Pull Request, I am requesting here. I hope it is okay. I will try the Pull Request again, tomorrow (Feb 3). Requests:

(1) Style Guide section: Update an erroneous link 'https://docs.gitlab.com/ee/development/documentation/styleguide.html#language' with a working link 'https://docs.gitlab.com/ee/development/documentation/styleguide/#documentation-style-guide'. 

(2) Building the Documentation App section: As Pavel pointed to the correct source, this request is to remove '/website' from the 'cd docs.webiny.com/website' command, as shown below.    

	1	git clone https://github.com/webiny/docs.webiny.com.git
	2	cd docs.webiny.com
	3	yarn
	4	yarn start

(3) When we use 'yarn start', an error message informs us that the Node.js version must be 14.x (14.18.3 is the latest, I suppose). Probably, we can alert users beforehand - 'Download Node.js version 14 or later'. 
Error message:  

...  $ yarn start
[ERROR] Minimum Node.js version not met :(
[INFO] You are using Node.js v12.22.9, Requirement: Node.js >=14

## Relevant Links

- [A change on X page](#)
- [Update list of libraries](#)
- [A new diagram with updated resources](#)

## Checklist
- [ ] I added page metadata (description, keywords)
- [ ] I've added "Can I Use This?" section (if needed, e.g. if documenting a new feature)
- [ ] I added `What You'll Learn` at the top of the page and every item in the list starts with a lower case letter
- [ ] I used title case for titles and subtitles (in the main menu and in the page content)
- [ ] I checked for typos and grammar mistakes
- [ ] I added `alt` / `title` attributes for inserted images (if any)
- [ ] When linking code from GitHub, I did it via a specific tag (and not `next` / `v5` branch) 

## Screenshots (if relevant):
N/A

### CODE REVIEW

File: docs/contributing/documentation.mdx [hunk 1]
```diff
@@ -54,9 +54,9 @@ Webiny docs repository uses [docusaurus](https://v2.docusaurus.io/) as the syste
...
 
 1. `git clone https://github.com/webiny/docs.webiny.com.git`
-2. `cd docs.webiny.com/website`
-3. `npm install`
-4. `npm start`
+2. `cd docs.webiny.com`
+3. `yarn`
+4. `yarn start`
 
 All the articles are inside the `/docs` folder. Edit `/website/sidebars.js` if you need to manage the items in the sidebar.
```
Comment (Pavel910) [inline docs/contributing/documentation.mdx:61]: `/website/sidebars.js` should just be `sidebars.js` as well.

**Review by Pavel910 — CHANGES_REQUESTED** [review]

File: docs/contributing/documentation.mdx (hunk 1 above)
Comment (pradeephubgit) [reply docs/contributing/documentation.mdx:61]: Ah, that's right, Pavel :). I missed that. 
I updated '/docs' folder to 'docs' folder. I hope it is okay?

Comment (Pavel910) [reply docs/contributing/documentation.mdx:61]: We don't have hard rule on that, the `/` just makes it  very clear that the `docs` folder is located in the root of the repository.

**Comment by pradeephubgit** [comment]
Hi Pavel, Others - Can you please advise if I should close this pull request, or someone with rights has to do this? 

Thanks.

**Comment by Pavel910** [comment]
@pradeephubgit what about the package.json / node version? Would you like to tackle that in this PR as well?

**Comment by pradeephubgit** [comment]
***@***.*** <https://github.com/pradeephubgit> what about the
package.json / node version? Would you like to tackle that in this PR as
well?>>

Hi Pavel - This is is something I am not clear about, and I wanted to ask
you about it in chat.
Is your suggestion that users can add the 'engines' property in their local
package.json file (corresponding to the npm folder)? If so, I will add a
note, likewise. Or was it something else? Can you clarify? Pls excuse me if
this is a dumb question :) .

Thanks,
Pradeep.

**Comment by Pavel910** [comment]
@pradeephubgit not really. Docusaurus requires node v14, and it will not let you start the local app:
image

For this I was thinking of adding the `engines` property to package.json, which is supposed to tell you that same thing when you run `yarn`. But it's not a super important thing, as Docusaurus does that check for you anyhow.

I'll merge this PR. Thanks for taking the time to contribute!

**Comment by pradeephubgit** [comment]
Hi Pavel - When I tried to install Vale, I had some issues. I will revert
if there is any value in sharing my experience, since it is not a
Webiny-related.
Thanks.

**Comment by Pavel910** [comment]
@pradeephubgit yeah Vale is annoying, feel free to post your experience, and if you solved the problem, post a solution. Thanks!

**Comment by pradeephubgit** [comment]
Hi Pavel - This my Vale installation experience. 'Vale_Installation.dox' contains the installation steps I used. Vale_Errors.docx contains the output of the 'vale docs' command. 

[Vale_Installation.docx](https://github.com/webiny/docs.webiny.com/files/8001358/Vale_Installation.docx)
[Vale_Errors.docx](https://github.com/webiny/docs.webiny.com/files/8001359/Vale_Errors.docx)

**Pull request merged**